### PR TITLE
Feat:Kanban & Task Details | Task Cards

### PR DESCRIPTION
**Pull request by rehanmohiuddin**
- Added Kanban
- Added Task Card
- Added Modal
- Added Create Task Comp
- Added Edit Task Comp

All UI's Are Done 

WIP:Context

### CODE REVIEW

File: src/Frontend/Components/Editor/index.jsx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    setEditorState(RichUtils.toggleInlineStyle(editorState, inlineStyle));
+  };
+
+  const handleBlockClick = (e, blockType) => {
+    e.preventDefault();
+    setEditorState(RichUtils.toggleBlockType(editorState, blockType));
+  };
+  const close = () => {};
+  //   console.log({ editorState });
+  //   console.log(convertToRaw(editorState.getCurrentContent()).blocks);
```
Comment (prathmeshjagtap) [inline src/Frontend/Components/Editor/index.jsx:67]: you can remove this commented code

File: src/Frontend/Components/TaskDetails/index.jsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    setEditorState(RichUtils.toggleInlineStyle(editorState, inlineStyle));
+  };
+
+  const handleBlockClick = (e, blockType) => {
+    e.preventDefault();
+    setEditorState(RichUtils.toggleBlockType(editorState, blockType));
+  };
+  const close = () => {};
+  //   console.log({ editorState });
+  //   console.log(convertToRaw(editorState.getCurrentContent()).blocks);
```
Comment (prathmeshjagtap) [inline src/Frontend/Components/TaskDetails/index.jsx:59]: you can remove this

File: src/Frontend/Components/TaskDetails/index.jsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+          </button>
+        </div>
+        <Editor
+          onChange={(e) => {
+            setEditorState(e);
+          }}
+          handleKeyCommand={handleKeyCommand}
+          editorState={EditorState
+            .createWithContent
+            // convertFromRaw(JSON.parse())
```
Comment (prathmeshjagtap) [inline src/Frontend/Components/TaskDetails/index.jsx:128]: you can remove commented  code if it is not needed in future

**Review by prathmeshjagtap — APPROVED** [review]
Code is Well Structured
UI is Clean 
Just work on some changes suggested

**Review by spraveenofficial — APPROVED** [review]

**Pull request merged**